gh-actions/go/generate/action.yaml

```diff
@@ -50,12 +54,17 @@ runs:
         tools=$(go mod edit --json | \
           jq -r '.Require[] | select(.Indirect!=true) | [.Path,.Version] | join("@")')
 
-        needsProtoc=false
         for tool in ${tools}; do
+          IFS='@' read -r -a pair <<< "${tool}"
+          echo "${pair[0]}"
           if [[ "${tool}" == *protoc* ]]; then
             echo "needs-protoc=true" >> $GITHUB_OUTPUT
           fi
-          go install ${tool}
+
+          cmd=$(grep -F "${pair[0]}" ./*.go | sed -nr 's/.*_\s+"([^"]+)".*/\1/p' | head -1)
```

Comment:
Add a command to gives examples against what this regexp against and how.

Reply:
The point of it is:
 1. Grep with regex disabled for the line containing the package URI
 2. Strip from a line such as `     _     "URI" ` the `URI`
 3. Ensure we have no duplicates.

By add a command what you mean? Add a comment with an example?

Reply:
Sorry, I meant "comment"